snuba/lw_deletions/strategy.py

```diff
@@ -73,7 +73,8 @@ def submit(self, message: Message[ValuesBatch[KafkaPayload]]) -> None:
     def _get_attribute_info(self) -> AttributionInfo:
         return AttributionInfo(
             app_id=AppID("lw-deletes"),
-            tenant_ids={},
+            # concurrent allocation policies requires project or org id
+            tenant_ids={"project_id": 1},
```

Comment:
Why 1?

Reply:
we are really just ratelimiting on the referrer, and so keeping the project hardcoded works here. Its the same thing we do in the subscriptions executor subscription. I can't pass in user project ids because the query will contain multiple projects and multiple organizations. And furthermore what we really want to limit is the number of concurrent mutations on the system